### PR TITLE
Handling multiple servers in plugin 

### LINKED ISSUE #97: Multiple servers handling in plugin
We need how-to for users who use multiple mergin servers or migrate from one to another.

### PR DESCRIPTION
**Pull request by varmar05**
PR adds how-to/setup for mergin plugin in case of using multiple servers.

closes #97

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ The ecosystem consist of various components:
 - [How to install the Mergin Maps plugin for QGIS](./setup/install-mergin-maps-plugin-for-qgis/)
 - [Working with Organisations](./setup/working-with-organisations/)
 - [Subscriptions and invoicing](./setup/subscriptions/)
+- [Use multiple servers in plugin](./setup/plugin-multi-server-use/)
```
Comment (PeterPetrik) [inline src/index.md:40]: The order is "Concepts, How-tos, References", can you move to how-to section.

Comment (PeterPetrik) [reply src/index.md:40]: Also name "How to use multiple servers in Mergin Maps Plugin for QGIS"

Maybe we should move it to section "Manage Account & Project ", as this is not about install/sign up, but about signing in...

Comment (PeterPetrik) [reply src/index.md:40]: for main menu , yuo need to add it to https://github.com/MerginMaps/docs/blob/main/src/.vuepress/sidebar/en.js

File: src/setup/plugin-multi-server-use/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# How to use MerginMaps plugin with multiple servers
```
Comment (PeterPetrik) [inline src/setup/plugin-multi-server-use/index.md:1]: Mergin Maps Plugin for QGIS

How to Use Mergin Maps Plugin for QGIS with Multiple Servers

File: src/setup/plugin-multi-server-use/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+There is a default [MerginMaps server](https://public.cloudmergin.co.uk) which is preconfigured in QGIS plugin. But there might be some cases you might want to use some custom server (e.g. you host it yourself). We will cover two typical use cases
+
+- migration from one server to another
+- using multiple servers simultaneously
+
+## Non-standard configuration
+
+Let's start with setting of custom Mergin server. It can be done in a configuration dialog
```
Comment (PeterPetrik) [inline src/setup/plugin-multi-server-use/index.md:10]: https://dev.merginmaps.com/docs/misc/write-docs/#naming-things

in plain text use https://github.com/MerginMaps/docs/tree/main/src/.vuepress/components

**Comment by PeterPetrik** [comment]
@alex-cit can you look at this PR? Thanks!

File: src/index.md [hunk 4]
```diff
@@ -44,11 +44,13 @@ The ecosystem consist of various components:
 - [How to create a new project](./manage/create-project/)
 - [Home page and dashboard](./manage/dashboard/)
 - [How to share, transfer or delete a project](./manage/project-advanced/)
+- [How to use multiple servers in the Mergin Maps plugin for QGIS](./manage/plugin-multi-server-use/)
```
Comment (alex-cit) [inline src/index.md:47]: This could be renamed to match the title of the page (How to Use Mergin Maps plugin with multiple servers)

**Review by alex-cit — APPROVED** [review]
Looking good!

**Pull request merged**